.github/workflows/ci-pipeline.yml

```
@@ -83,23 +89,13 @@ jobs:
       - name: Publish Docker image to Docker Hub
         run: docker push ${{ secrets.DOCKERHUB_USERNAME }}/todo-app:latest
 
-  heroku:
+  azure:
     needs: docker
-    name: Publish to Heroku
+    name: Publish to Azure Web App
     runs-on: ubuntu-latest
-    env:
-      HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
 
     steps:
       - uses: actions/checkout@v2         
 
-      - name: Pull and tag Docker image
-        run: |
-          heroku container:login
-          docker pull ${{ secrets.DOCKERHUB_USERNAME }}/todo-app:latest
-          docker tag ${{ secrets.DOCKERHUB_USERNAME }}/todo-app:latest registry.heroku.com/swish-todoapp/web
-
-      - name: Publish to Heroku
-        run: |
-          docker push registry.heroku.com/swish-todoapp/web
-          heroku container:release web -a swish-todoapp
\ No newline at end of file
+      - name: Publish changes to Azure Web App
+        run: curl -dH -X POST ${{ secrets.AZURE_WEBAPP_WEBHOOK_URL }}
```

It's good to add `--fail` in here, so that if Azure response with an error the job fails (curl considers any response a success by default, including 4xx and 5xx responses).

```suggestion
        run: curl --fail -dH -X POST ${{ secrets.AZURE_WEBAPP_WEBHOOK_URL }}
```